flowzone.yml

```diff
@@ -1634,7 +1645,7 @@ jobs:
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           accountId: ${{ secrets.CF_ACCOUNT_ID }}
-          command: pages publish --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} build/ >> $GITHUB_STEP_SUMMARY
+          command: pages publish --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} $GITHUB_WORKSPACE/build/ >> $GITHUB_STEP_SUMMARY
```

Comment:
```suggestion
          command: pages publish --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} ${{ inputs.working_directory }}/build/ >> $GITHUB_STEP_SUMMARY
```

Reply:
Yeah that would be better if working_directory input defaults to something sensible.

Reply:
Yeah, this won't work. It fails with: 

`✘ [ERROR] ENOENT: no such file or directory, scandir '/github/workspace/tests/build'`

This could be because we change directories to build the docs in a clean directory. Reverting this back to the original.